routing/routing.md

```diff
@@ -0,0 +1,57 @@
+# Routing
+
+This document depicts important information about configuring routing for an existing WSO2 product deployment in a
+ container platform.
+ 
+The following will be discussed in detail in the document.
+
+* [Exposing your application to outside](#exposing-your-application-to-outside )
+* [Configuring hostname](#configuring-hostname)
+* [DNS mapping](#dns-mapping)
+* [Configuring SSL](#configuring-ssl)
+* [Tips on Session Affinity](#tips-on-session-affinity)
+
+### Exposing your application to outside
+
+We recommend the use of [official WSO2 product Helm Charts](https://hub.helm.sh/charts/wso2) for production grade WSO2 
+product deployments in Kubernetes environments. [Kubernetes Ingress resources](https://kubernetes.io/docs/concepts/services-networking/ingress/)
+are used to expose WSO2 product deployments in Kubernetes environment. Kubernetes Ingress resources are capable of
+determining traffic based on the host and path forwarded to them by backends. It also operates on the application
+layer of the network stack (HTTP) and can provide features like cookie-based session affinity and SSL terminations.
+WSO2 recommend [NGINX])(https://kubernetes.github.io/ingress-nginx/) as the Ingress controller for production grade 
```

Comment:
To make it to a link change like this:
[NGINX](https://kubernetes.github.io/ingress-nginx/)

Reply:
Done